### PR TITLE
Update links to point to the Internet Archive for issue2799 and issue5726

### LINKED ISSUE #6854: Make linked test cases point to the Internet Archive
We have quite a number of linked test cases in the https://github.com/mozilla/pdf.js/tree/master/test/pdfs folder. Some of them point to sources other than the Internet Archive or Bugzilla and might become unavailable, which is to be avoided.

It is better to fetch the files from the Internet Archive to be sure that they remain available. This should always be possible if the current source exists because once you input the link on the Internet Archive website it will archive the file automatically.

Below is a list of linked test cases that point to external sources and should be updated (obtained with `grep -L "bugzilla.mozilla.org\|archive.org" *.pdf.link`). Make sure that the hash of the Internet Archive file is the same as the hash in the test manifest.

- [x] 20130226130259.pdf.link
- [x] JST2007-5.pdf.link
- [x] P020121130574743273239.pdf.link
- [x] SFAA_Japanese.pdf.link
- [x] TaroUTR50SortedList112.pdf.link
- [x] aboutstacks.pdf.link
- [x] artofwar.pdf.link
- [x] bpl13210.pdf.link
- [x] bug766138.pdf.link
- [x] bug808084.pdf.link
- [x] bug887152.pdf.link
- [x] cable.pdf.link
- [x] ecma262.pdf.link
- [x] fips197.pdf.link
- [x] fit11-talk.pdf.link
- [x] geothermal.pdf.link
- [x] hmm.pdf.link
- [x] html5checker.pdf.link
- [x] hudsonsurvey.pdf.link
- [x] ichiji.pdf.link
- [x] issue1010.pdf.link
- [x] issue1015.pdf.link
- [x] issue1049.pdf.link
- [x] issue1096.pdf.link
- [x] issue1127.pdf.link
- [x] issue1133.pdf.link
- [x] issue1155.pdf.link
- [x] issue1169.pdf.link
- [x] issue1233.pdf.link
- [x] issue1257.pdf.link
- [x] issue1309.pdf.link
- [x] issue1317.pdf.link
- [x] issue1419.pdf.link
- [x] issue1466.pdf.link
- [x] issue1597.pdf.link
- [x] issue1629.pdf.link
- [x] issue1658.pdf.link
- [x] issue1685.pdf.link
- [x] issue1687.pdf.link
- [x] issue1709.pdf.link
- [x] issue1721.pdf.link
- [x] issue1729.pdf.link
- [x] issue1796.pdf.link
- [x] issue1810.pdf.link
- [x] issue1878.pdf.link
- [x] issue1912.pdf.link
- [x] issue1936.pdf.link
- [x] issue1998.pdf.link
- [x] issue2006.pdf.link
- [x] issue2129.pdf.link
- [x] issue2139.pdf.link
- [x] issue2386.pdf.link
- [x] issue2442.pdf.link
- [x] issue2531.pdf.link
- [x] issue2537.pdf.link
- [x] issue2627.pdf.link
- [ ] issue2799.pdf.link
- [x] issue2829.pdf.link
- [x] issue2853.pdf.link
- [x] issue2881.pdf.link
- [x] issue3062.pdf.link
- [x] issue3384.pdf.link
- [x] issue3666.pdf.link
- [x] issue3848.pdf.link
- [x] issue3903.pdf.link
- [x] issue3925.pdf.link
- [x] issue3999.pdf.link
- [x] issue4926.pdf.link
- [x] issue5592.pdf.link
- [ ] issue5726.pdf.link
- [x] issue818.pdf.link
- [x] issue919.pdf.link
- [x] jai.pdf.link
- [x] kdchart.pdf.link
- [x] liveprogramming.pdf.link
- [x] mao.pdf.link
- [x] ocs.pdf.link
- [x] ohkubo-SS04.pdf.link
- [x] pal-o47.pdf.link
- [x] pdf.pdf.link
- [x] piperine.pdf.link
- [x] preistabelle.pdf.link
- [x] protectip.pdf.link
- [x] tcpdf_033.pdf.link
- [x] tutorial.pdf.link
- [x] txt2pdf.pdf.link
- [x] unix01.pdf.link
- [x] usmanm-bad.pdf.link
- [x] vesta.pdf.link
- [ ] yo01.pdf.link

Additionally:
- [x] Investigate `hmm.pdf`. The MD5 hash in the manifest matches the MD5 hash of the file from the link, but the bots complain about a mismatch.

### PR DESCRIPTION
**Pull request by timvandermeij**
We remove yo01. The file is not available, it was introduced in https://github.com/mozilla/pdf.js/commit/3529658ac5a6aa2ca137d8f77ab1470af43abbab along with another test file (so we should not need it) and it was replaced in https://github.com/mozilla/pdf.js/commit/6b2c6fc22397350ace1c65867e33b9b02c1fdc4b, so it was only in the repository for a very short time.

Fixes #6854 and #5091. All test cases are now either reduced or link to trusted sources like Bugzilla and the Internet Archive, making sure that they keep on existing and do not change. MD5 mismatches have also been resolved.

### CODE REVIEW

**Comment by Snuffleupagus** [comment]
> We remove yo01. The file is not available, it was introduced in 3529658 along with another test file (so we should not need it) and it was replaced in 6b2c6fc, so it was only in the repository for a very short time.

I'm not sure if I agree with this conclusion, since I seen no indication that `usmanm-bad-auto-fetch` was intended to *replace* `yo01`. It rather seems to me that it's more likely that, given that it was already unavailable at the the time of that patch, it was might have been removed a little to hastily.

I'm assuming that the two files added in 3529658 doesn't demonstrate entirely identical issues, since otherwise I cannot image why both were added. @timvandermeij Did you verify that both those files failed in *the same* way previously?
(I've been meaning to replace `yo01` with a reduced test-case, but I've not yet had the time).

**Comment by timvandermeij** [comment]
I should have been a bit more clear indeed. I tested both files from https://github.com/mozilla/pdf.js/commit/3529658ac5a6aa2ca137d8f77ab1470af43abbab (the unavailable file substituted by https://web.archive.org/web/20120622065546/http://www.ugokotsu.co.jp/ug/rosen/jikoku/yo01.pdf) with the previews from https://github.com/mozilla/pdf.js/pull/2839 (just before the fix, both broken because characters are shifted to the right) and https://github.com/mozilla/pdf.js/pull/2848 (just after the fix, both fixed because the characters are in the right position). That is how I came to my conclusion, but it could be that I have missed something that you do notice! In that case, please let me know. We can then do two things: either remove the file, or I'll leave yo01 out of this comment so you can replace it with a reduced file.

**Comment by Snuffleupagus** [comment]
OK, I've just tested this using the "right" version of `yo01.pdf` (I still have a copy locally), and it appears that the failures are the same as with `20130226130259.pdf`.
Looking at issue #2842, it specifically mentions Windows XP (which I'm not using), but it does seem *highly* likely that those two files demonstrate the *same* issue. And I suppose you do have a good point about that test being very short-lived!

So in summary, I'm fine with just removing the link (mean less work ;-), so I'll test/verify this locally later tonight and then merge it. Thanks for providing additional context here!

**Comment by Snuffleupagus** [comment]
Looks good, thank you!
